packages/angular_devkit/build_ng_packagr/package.json

```diff
@@ -9,17 +9,15 @@
   "dependencies": {
     "@angular-devkit/architect": "0.0.0",
     "@angular-devkit/core": "0.0.0",
-    "rxjs": "6.4.0",
-    "semver": "5.6.0"
+    "rxjs": "6.4.0"
   },
   "peerDependencies": {
-    "ng-packagr": "^4.0.0"
+    "ng-packagr": "^4.0.0 || 5.0.0"
```

Comment:
peer dependencies should be a range (`^5.0.0`).

Reply:
Oops